api/BUILD

```diff
@@ -88,6 +88,8 @@ proto_library(
         "//envoy/config/rbac/v3alpha:pkg",
         "//envoy/config/resource_monitor/fixed_heap/v2alpha:pkg",
         "//envoy/config/resource_monitor/injected_resource/v2alpha:pkg",
+        "//envoy/config/retry/omit_hosts/v2:pkg",
+        "//envoy/config/retry/omit_hosts/v3alpha:pkg",
```

Comment:
The `v3alpha` APIs should now live under `envoy.extensions`.

Reply:
@htuch the format check is failing if I place v3alpha under //envoy/extensions. Please advise?

Reply:
Take a look at https://github.com/envoyproxy/envoy/blob/master/api/envoy/config/filter/network/rate_limit/v2/rate_limit.proto#L16 for how this can be done. You need an annotation in the v2 package to drive th e tooling to migrate to v3alpha.

Reply:
Thanks! I overlooked package name change in one of the files which resulted in an error. Fixed now.